README.md

```diff
@@ -1,8 +1,10 @@
 <!--
-This file was generate by MarkdownSnippets.
+GENERATED FILE - DO NOT EDIT
+This file was generated by [MarkdownSnippets](https://github.com/SimonCropp/MarkdownSnippets).
 Source File: /README.source.md
-To change this file edit the source file and then re-run the generation using either the dotnet global tool (https://github.com/SimonCropp/MarkdownSnippets#markdownsnippetstool) or using the api (https://github.com/SimonCropp/MarkdownSnippets#running-as-a-unit-test).
+To change this file edit the source file and then run MarkdownSnippets.
```

Comment:
I still like the unit test approach more (see comment below), but wonder what's the benefit of the .NET global tool over unit test.

Reply:
>  what's the benefit of the .NET global tool over unit test.

i didnt say there was. however i think there is a benefit of using msbuild over unit tests.